vmdk_plugin/Makefile

```diff
@@ -101,7 +101,9 @@ SRC = main.go log_formatter.go utils/refcount/refcnt.go \
 
 TEST_SRC = ../tests/utils/inputparams/testparams.go \
 	../tests/utils/dockercli/volumelifecycle.go \
-	../tests/utils/verification/volumeproperties.go
+	../tests/utils/verification/volumeproperties.go \
```

Comment:
as far as I remember .. IIRC  `/tests/utils/inputparams/testparams.go` is the only need .. can you verify by adjusting .. keep only the required one and knock of irrelevant.